pkg/gui/controllers/helpers/refresh_helper.go

```diff
@@ -249,6 +249,32 @@ func (self *RefreshHelper) refreshCommits() {
 	wg.Wait()
 }
 
+func (self *RefreshHelper) determineCheckoutBranchName() string {
+	if rebasedBranch := self.c.Git().Status.BranchBeingRebased(); rebasedBranch != "" {
+		// During a rebase we're on a detached head, so cannot determine the
+		// branch name in the usual way. We need to read it from the
+		// ".git/rebase-merge/head-name" file instead.
+		return strings.TrimPrefix(rebasedBranch, "refs/heads/")
+	}
+
+	if bisectedBranch := self.c.Git().Bisect.GetInfo().GetStartSha(); bisectedBranch != "" {
```

Comment:
Just noticed that the StartSha is already set right after the first `b g` or `b b`, before we're actually on a detached head. I think we also need to check either `Started()` or `Bisecting()` but I'm not sure which one (I don't really get the difference).

Reply:
I'd use `Bisecting()` because it's only in the bisecting phase where we have a detached head. `Started()` is for when you've marked one commit as good/bad but you haven't marked the other commit yet, meaning git hasn't started jumping to different commits.

Reply:
OK, changed in 182b5a8075c8